daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/debugger/InteractiveDebugger.scala

```diff
@@ -138,8 +138,10 @@ class InteractiveDebugger(
     /* keeps track of which parse step we're on for trace output */
     var parseStep = 0
 
+    import org.apache.daffodil.lib.schema.annotation.props.Enum
     /* how to display data */
-    var representation: Representation.Value = Representation.Text
+    var representation: Enum.Value[Representation] =
+      Representation.Text
```

Comment:
Maybe this is the reason for needing the type parameter in Enum.Value, mentioned in a previous comment? Though, could this be changed to this:
```scala
var representation: Representation = Representation.Text
```

`Representation` is a trait that all representation enum values extend so I think this type is still accurate? It's maybe less clear that `Representation` is an enumeration value, but it's definitely cleaner. In fact, I *think* this is the only place where we use `Foo.Value` to reference our custom enum types since this is the only one you had to change.